hedy.py

```diff
@@ -2733,6 +2755,8 @@ def is_program_valid(program_root, input_string, level, lang):
             raise exceptions.UnsupportedFloatException(value=''.join(invalid_info.arguments))
         elif invalid_info.error_type == 'lonely text':
             raise exceptions.LonelyTextException(level=level, line_number=line)
+        elif invalid_info.error_type == 'invalid random keyword':
```

Comment:
I dont believe this is needed anymore since I removed the rule from the lark file